relay-metrics/src/protocol.rs

```diff
@@ -152,20 +152,41 @@ fn is_valid_name(name: &str) -> bool {
     false
 }
 
-/// Validates an MRI of the form `<ty>:<ns>/<name>@<unit>`
-///
-/// Note that the format used in the statsd protocol is different: Metric names are not prefixed
-/// with `<ty>:` as the type is somewhere else in the protocol.
-pub(crate) fn is_valid_mri(name: &str) -> bool {
-    let mut components = name.splitn(2, ':');
-    if components.next().is_none() {
-        return false;
-    }
+/// A metric name parsed as MRI, a naming scheme which includes most of the metric's bucket key
+/// (excl. timestamp and tags).
+pub struct MetricMri {
```

Comment:
MRI stands for "MetricResourceIdentifier", so the "MetricMri" is redundant. The acronym is unique enough to just call it `Mri`, or fully type it out.

Reply:
If the name is refactored, I rather `MetricResourceIdentifier` over `Mri`.